templates/navbar-for-user.html

```diff
@@ -18,7 +18,7 @@
                     <div class="title-plus-arrow">
                         <a class="n-right" href="#"><%= userName %></a>
```

Comment:
some ejs stuff right here  <%= userName %> 
should be removed

Reply:
I should actually delete this file.

Reply:
Deleted, closing comment.